README, script, tfvars

vicenteg opened the pull request:
Add a README

Update script with some error checking, and to look for the current branch.

Update the terraform with variables, and to allow the user to supply the state backend details.

vicenteg committed:
README, script, tfvars

Add a README

Update script with some error checking, and to look for the
current branch.

Update the terraform with variables, and to allow the user
to supply the state backend details.

vicenteg committed:
adding changes to backend, datasets

vicenteg committed:
change bucket name

danieldeleo commented on run_terraform.sh:
```diff
@@ -1,11 +1,27 @@
-cd terraform
+#!/bin/sh -ex
+
+if [ -z "${BRANCH_NAME}" ]; then
+  BRANCH_NAME=$(git branch --show-current)
+fi
+echo "Using '${BRANCH_NAME}' environment"
```
Cloud Build provides BRANCH_NAME as a [default substitution variable](https://cloud.google.com/build/docs/configuring-builds/substitute-variable-values#using_default_substitutions). Are you proposing this change for a use-case outside of Cloud Build?

vicenteg replied:
Oh, given the shell script I assumed this could be run interactively.

danieldeleo replied:
In a real world scenario, you wouldn't want humans running it interactively. The [cloudbuild.yaml](https://github.com/danieldeleo/bigquerycicd/blob/prod/cloudbuild.yaml) file is what invokes the run_terraform.sh script

vicenteg committed:
Delete run_terraform.sh

removing changes to run_terraform.sh, since the script is intended only to be run from Cloud Build triggers and not manually via `gcloud builds submit`.
